source/features/show-names.tsx

```diff
@@ -54,6 +52,11 @@ async function updateLink(batchedUsernameElements: HTMLAnchorElement[]): Promise
 		// If it's a regular comment author, add it outside <strong> otherwise it's something like "User added some commits"
 		const {parentElement} = usernameElement;
 		const insertionPoint = parentElement!.tagName === 'STRONG' ? parentElement! : usernameElement;
+
+		// This change is ideal but should not break the feature if it fails
+		// And it should be done before inserting the name
+		dropExtraCopy(usernameElement);
```

Comment:
This chance goes against the previous comment: it should be last specifically to avoid breaking the entire feature when this piece breaks. 

If you prefer, update the comment and wrap this call in a try/catch (+ `feature.log.error` call in case of errors)